src/main/java/domain/table/order/Orders.java

```diff
@@ -0,0 +1,64 @@
+package domain.table.order;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+public class Orders {
+    private static final int DISCOUNT_UNIT = 10;
+    private static final int DISCOUNT_PRICE_PER_UNIT = 10000;
+
+    private final List<Order> orders = new ArrayList<>();
+
+    public void add(final Order additionalOrder) {
+        Optional<Order> maybeOrder = findOrderByMenu(additionalOrder);
+        if (maybeOrder.isPresent()) {
+            maybeOrder.ifPresent(order -> order.addAmount(additionalOrder.getOrderAmount()));
+        } else {
+            orders.add(additionalOrder);
+        }
+    }
+
+    public long calculateTotalPrice() {
+        long totalPrice = sumAllPrice();
+        return discountByChickenCount(totalPrice);
+    }
+
+    private long sumAllPrice() {
+        return orders.stream()
+                .mapToLong(Order::calculateTotalPrice)
+                .sum();
+    }
+
+    private long discountByChickenCount(final long totalPrice) {
+        int chickenCount = calculateTotalChickenCount();
+        long discountPrice = chickenCount / DISCOUNT_UNIT * DISCOUNT_PRICE_PER_UNIT;
+        return totalPrice - discountPrice;
+    }
+
+    private int calculateTotalChickenCount() {
+        return orders.stream()
+                .filter(Order::isChicken)
+                .mapToInt(order -> order.getOrderAmount().getAmount())
```

Comment:
디미터의 법칙을 어기신거 같은데...? 방법이 없을까요!?

Reply:
`getOrderAmount()`가 int를 반환하도록 하자니 다른 곳에서 쓰일 가능성을 생각하면 바꾸기 애매하네요.

```suggestion
                .map(order -> order.getOrderAmount())
                .map(orderAmount -> orderAmount.getAmount())
```
디미터 법칙을 지키고자 위와같이 하는건 불필요한 추가 mapping이라고 판단하여 일단은 놔뒀습니다. 더 좋은 해결책을 가지고 계시다면 조언 부탁드립니다!